repos/keg-components/src/components/form/checkbox/checkbox.js

```diff
@@ -45,53 +44,70 @@ const Input = StyleInjector(KegInput, {
   className: 'keg-checkbox'
 })
 
-const Element = React.forwardRef(
-  (
-    { className, elProps, styles = noPropObj, icon, checked, ...props },
-    ref
-  ) => {
-    const checkStyle = useMemo(() => {
-      return {
-        ...styles.indicator,
-        ...checkBoxStyles.icon,
-      }
-    }, [ checkBoxStyles, styles ])
+const Element = React.forwardRef((props, ref) => {
+  const {
+    className,
+    elProps,
+    styles = noPropObj,
+    CheckIcon = Check,
+    checked,
+    ...attributes
+  } = props
+
+  const checkStyle = useMemo(() => {
+    return {
+      ...checkBoxStyles.icon,
```

Comment:
Since I'm using a custom icon, I needed to style it differently (mainly width and height) to match the events force mock up.

But why should they not be overwritten? I get that in 99% of cases, the styles need to stay this way otherwise things look wrong. But haven't we always said that we want the styles to be fully configurable, because of how frustrating it is when other component libraries don't allow that?

I understand why the `input` styles should be locked, because the input isn't actually visible at all and exists solely for the functionality of the component. I'll definitely revert that change (I'm not overwriting the input styles).
 
But the icon is visible and I think it makes sense to let consumers style it if they need to, no?